monai/data/image_reader.py

```diff
@@ -833,8 +834,13 @@ def get_data(
         metadata["spatial_shape"] = np.asarray(region.shape[:-1])
         metadata["original_channel_dim"] = -1
 
+        # combine image and metadata
+        if not isinstance(region, MetaTensor):
```

Comment:
I think this wsi_reader is deprecated, maybe no need to update it?
We should update this part instead:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/wsi_reader.py#L234
@drbeh Could you please help double confirm it?

Thanks in advance.

Reply:
That's correct! `WSIReader` in `image_reader.py` is going to be deprecated in favor of the one in `wsi_reader.py`. Thanks